app/services/schools/bill_request_service.rb

```diff
@@ -0,0 +1,18 @@
+module Schools
+  class BillRequestService
+    def initialize(school)
+      @school = school
+    end
+
+    def users
+      users = @school.users.school_admin + @school.users.staff
+      users.sort_by(&:staff_role)
+    end
+
+    def request_documentation!(users, meters = [])
+      electricity_meters = meters.select {|m| m.meter_type.to_s == "electricity"}
```

Comment:
I think you can use the enum types as a boolean to select meters, e.g.
electricity_meters = meters.select(&:electricity?)
gas_meters = meters.select(&:gas?)